<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewWindow</class>
 <widget class="QMainWindow" name="ViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView{
border-width:2px
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Delete"/>
   <addaction name="separator"/>
   <addaction name="action_Edit"/>
   <addaction name="separator"/>
   <addaction name="action_Yes"/>
   <addaction name="action_No"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="icons/ViewWindowRes.qrc">
     <normaloff>:/plus.png</normaloff>:/plus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="icon">
    <iconset resource="icons/ViewWindowRes.qrc">
     <normaloff>:/trash.png</normaloff>:/trash.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="icon">
    <iconset resource="icons/ViewWindowRes.qrc">
     <normaloff>:/note.png</normaloff>:/note.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="action_Yes">
   <property name="icon">
    <iconset resource="icons/ViewWindowRes.qrc">
     <normaloff>:/check.png</normaloff>:/check.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Yes</string>
   </property>
  </action>
  <action name="action_No">
   <property name="icon">
    <iconset resource="icons/ViewWindowRes.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;No</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/ViewWindowRes.qrc"/>
 </resources>
 <connections/>
</ui>
